src/ddsrt/include/dds/ddsrt/random.h

```diff
@@ -11,6 +11,15 @@
 #ifndef DDSRT_RANDOM_H
 #define DDSRT_RANDOM_H
 
+/**
+ * @file random.h
+ * 
+ * Pseudo random number generator known as the Mersenne Twister.
+ * It generates uint32_t from a uniform distribution in the range 0 to 0xffffffff (the maximum value of uint32_t).
+ * A useful property is that the use of a fixed seed allows you to reproduce
```

Comment:
This is true, but it is also the very definition of a **P**RNG and, I would hope, common knowledge amongst all software developers. So in my view, it should not be in the documentation.

Given that there are a few other changes that I would like to see, perhaps you can remove this, too.